### PR TITLE
[Tizen/Native] upload sample app of nnstreamer single API @open sesame 11/14 10:26 

### PR DESCRIPTION
**Pull request by helloahn**
This is a simple example of nnstreamer single API
To run this app properly, the [PR](https://github.com/nnsuite/nnstreamer/pull/1860) have to be merged.

Signed-off-by: HyoungJoo Ahn <hello.ahn@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: Tizen.native/ImageClassification/inc/main.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
```
Comment (myungjoo) [inline Tizen.native/ImageClassification/inc/main.h]: We don't use LGPL for apps. Apache-2.0 is recommended.

File: Tizen.native/ImageClassification/inc/view.h [hunk 2]
```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
```
Comment (myungjoo) [inline Tizen.native/ImageClassification/inc/view.h]: Same here. Unless you have some other intention, please use Apache-2.0 for apps.

File: Tizen.native/ImageClassification/src/main.c [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+  GError *err = NULL;
+  gchar *contents = NULL;
+  char *res_path = app_get_resource_path ();
+  gchar * label_path = g_strdup_printf ("%s/labels.txt", res_path);
+
+  if (!g_file_get_contents (label_path, &contents, NULL, &err)) {
+    dlog_print (DLOG_ERROR, LOG_TAG, "Unable to read file %s with error %s.",
+        label_path, err->message);
+    g_clear_error (&err);
+    return;
```
Comment (jaeyun-jung) [inline Tizen.native/ImageClassification/src/main.c]: need to free path strings if failed to get file content.
```
if (g_file_get_contents()) {
  labels = g_strsplit (contents, "\n", -1);
  g_free (contents);
} else {
  print log, clear_error
}

g_free (label_path);
free (res_path);
```

File: Tizen.native/ImageClassification/src/data.c [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+    case CAMERA_STATE_CAPTURED:
+      return "CAMERA_STATE_CAPTURED";
+
+    default:
+      return "Unknown";
+  }
+}
+
+/**
+ * @brief Sets the camera brightness.
```
Comment (jaeyun-jung) [inline Tizen.native/ImageClassification/src/data.c]: plz update description and details (not brightness btn)

File: Tizen.native/ImageClassification/src/main.c (hunk 3 above)
Comment (helloahn) [reply Tizen.native/ImageClassification/src/main.c]: it's updated

File: Tizen.native/ImageClassification/src/data.c (hunk 4 above)
Comment (helloahn) [reply Tizen.native/ImageClassification/src/data.c]: it's updated

**Pull request merged**